packages/calypso-e2e/src/lib/components/editor-gutenberg-component.ts

```diff
@@ -58,7 +58,7 @@ export class EditorGutenbergComponent {
 	async enterTitle( title: string ): Promise< void > {
 		const sanitizedTitle = title.trim();
 		const locator = this.editor.locator( selectors.title );
-		await locator.fill( sanitizedTitle, { timeout: 20 * 1000 } );
```

Comment:
Did that for debugging purposes, should be reverted later if this ever gets accepted.